http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/package.scala

```diff
@@ -82,4 +85,33 @@ package object service {
         )
       )
 
+  private[service] def processCreatorOnlyParameter(
+    userInfo: UserInfo,
+    params: LabelMap
+  ): Either[ParseCreatorOnlyException, Option[WorkbenchEmail]] =
+    // Support filtering by creator either by role=creator query string, or creator=<user email> label
```

Comment:
super small nit, but could you also add in the comment the preferred method and why to disseminate the context shared by you and Qi in the PR comments?

Reply:
yes!